tests/sql_mode/run.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
```

Comment:
New test case should be added in other_intergration.txt

Reply:
done